README.md

```diff
@@ -26,157 +26,8 @@ For the purposes of complying with our code license, you can use the following S
-	}
-	os.Stdout.Write(jsonbuf)
-}
-```
-
-While you can directly put the output of this programme into the file mentioned above, you can also remove the public
-key from the set and change the `kid` of the private key to not include the prefix `private:`.
-
-### CockroachDB Setup
-
-Kratos uses CockroachDB to store its data. For that data to be shared across all nodes that comprise your portal cluster
-setup, we need to set up a CockroachDB cluster, complete with secure communication.
-
-#### Generate the certificates for secure communication
-
-For a detailed walk-through, please check [this guide](https://www.cockroachlabs.com/docs/v20.2/secure-a-cluster.html)
-out.
-
-Steps:
-
-1. Start a local cockroach docker instance:
-   `docker run -d -v "<local dir>:/cockroach/cockroach-secure" --name=crdb cockroachdb/cockroach start --insecure`
-1. Get a shall into that instance: `docker exec -it crdb /bin/bash`
-1. Go to the directory we which we mapped to a local dir: `cd /cockroach/cockroach-secure`
-1. Create the subdirectories in which to create certificates and keys: `mkdir certs my-safe-directory`
-1. Create the CA (Certificate Authority) certificate and key
-   pair: `cockroach cert create-ca --certs-dir=certs --ca-key=my-safe-directory/ca.key`
-1. Create a client certificate and key pair for the root
-   user: `cockroach cert create-client root --certs-dir=certs --ca-key=my-safe-directory/ca.key`
-1. Create the certificate and key pair for your
-   nodes: `cockroach cert create-node cockroach mynode.siasky.net --certs-dir=certs --ca-key=my-safe-directory/ca.key`.
-   Don't forget the `cockroach` node name - it's needed by our docker-compose setup. If you want to create certificates
-   for more nodes, just delete the `node.*` files (after you've finished the next steps for this node!) and re-run the
-   above command with the new node name.
-1. Put the contents of the `certs` folder under `docker/cockroach/certs/*` under your portal's root dir and store the
-   content of `my-safe-directory` somewhere safe.
-1. Put _another copy_ of those certificates under `docker/kratos/cr_certs` and change permissions of the `*.key` files,
-   so they can be read by anyone (644).
-
-#### Configure your CockroachDB node
-
-Open port 26257 on all nodes that will take part in the cluster. Ideally, you would only open the port for the other
-nodes in the cluster.
-
-There is some configuration that needs to be added to your `.env`file, namely:
-
-1. CR_IP - the public IP of your node
-1. CR_CLUSTER_NODES - a list of IPs and ports which make up your cluster, e.g.
-   `95.216.13.185:26257,147.135.37.21:26257,144.76.136.122:26257`. This will be the list of nodes that will make up your
-   cluster, so make sure those are accurate.
+## Running a Portal
+For those interested in running a Webportal, head over to our developer docs [here](https://docs.siasky.net/developer-guides/operating-a-skynet-webportal) to learn more.
```

Comment:
I don't really like this transition. We're moving from clear, direct instructions on how to set up and start running the portal to a vague guide on how to generally run a portal. The information is partially there, for people who manage to find it under https://docs.siasky.net/webportal-management/setup/cluster-setup/accounts but it's also not completely there because the Mongo section is not yet available.

I am raising this objection mainly because this is something I personally find problematic with projects' documentation - if a project has clear and direct instructions on how to set it up and use it right in its readme, I am much more likely to tinker with it and to try it out than if it directs me to a general and scattered documentation portal. My aim is not to criticize how our docs portal is structured (I believe that this structure works for certain people) but to point out that this de-duplication of documentation might be a straight up negative change for some people.

Reply:
Since this original comment, at least 2 community members have tried to use the scripts in this repo and it has broken their servers. We are not maintaining the setup scripts in this repo and need to remove documentation that encourages users to use set up processes outside of our maintained documentation.

Reply:
Given that we've massively improved our handbook and we want to make it the official way for people to set up their systems, I believe it's time to merge this.